src/ape/api/networks.py

```diff
@@ -257,11 +258,7 @@ def providers(self):  # -> Dict[str, Partial[ProviderAPI]]
                     request_header=self.request_header,
                 )
 
-        if len(providers) > 0:
-            return providers
-
-        else:
-            raise NetworkError("No network providers found")
+        return providers
```

Comment:
`mainnet-fork` network is empty until you add a plugin like `ape-hardhat`

Reply:
See above comment about adding new networks